package.json

```diff
@@ -5,18 +5,22 @@
 	"author": "ISV SWAT",
 	"bugs": "https://github.com/forcedotcom/sfdx-scanner/issues",
 	"dependencies": {
-		"@lwc/eslint-plugin-lwc": "^0.10.0",
+		"@babel/core": "^7.11.0",
+		"@babel/eslint-parser": "^7",
+		"@lwc/eslint-plugin-lwc": "~1.1.2",
 		"@oclif/command": "^1",
 		"@oclif/config": "^1",
 		"@oclif/errors": "^1",
 		"@salesforce/command": "^4",
 		"@salesforce/core": "^2.1.6",
-		"@salesforce/eslint-config-lwc": "^0.7.0",
-		"@typescript-eslint/eslint-plugin": "^2.21.0",
-		"@typescript-eslint/parser": "^2.21.0",
-		"babel-eslint": "^10.1.0",
+		"@salesforce/eslint-config-lwc": "~3.2.1",
```

Comment:
16, 17, 22, and 23 are peer dependencies for the latest LWC.

Reply:
Can we use `^` everywhere so that we don't have to add a release activity to check for the latest?